src/XIVLauncher.Core/Components/SettingsPage/Tabs/SettingsTabWine.cs

```diff
@@ -51,7 +52,10 @@ public SettingsTabWine()
                     return null;
                 }
             },
-
+            new SettingsEntry<string>("DXVK Frame Limit", "Configure how many frames DXVK should be limited to. Set to 0 for unlimited.", () => Program.Config.DxvkFrameRate ?? "0", s => Program.Config.DxvkFrameRate = s)
+            {
+                CheckValidity = s => Regex.IsMatch(s, @"^\d{1,3}$") ? null : "Please specify a valid integer below 999."
```

Comment:
Regex is a bit overkill to check for a non-negative integer; afaik dxvk can also limit to more than 999fps